### ISSUE TITLE
<RadioGroup/> puppeteer testkit exceptoion

### ISSUE THREAD

**Issue by alexandraseri** [wix/wix-style-react#5811]
# 🐛 Bug Report

### 🏗 Relevant Components

`<RadioGroup />` puppeteer testkit

...

### 😯 Current Behavior

When trying to use `<RadioGroup />` puppeteer testkit, an exception is thrown:

```
import { radioGroupTestkitFactory } from 'wix-style-react/dist/testkit/puppeteer';
​
//in test
const radioGroupDriver = await radioGroupTestkitFactory({ dataHook, page });
​
​
/// causes error
TypeError: driverFactory is not a function
```

### 🤔 Expected Behavior

`<RadioGroup />` testkit works just like all others

### 👣 Steps to Reproduce

create puppeteer Box testkit in e2e

- WSR version: 8.39.0
- Yoshi version: 4.139.0

### 👀 Severity

Low

**Comment by shlomitc** [comment]
I think this is related to the fact that we export multiple functions exported from the driver. [This function export](https://github.com/wix/wix-style-react/blob/739b5d7f7569779923d482aa37aee3d7245c4f28/src/RadioGroup/RadioGroup.uni.driver.js#L5) is causing the pain
Our automated tools generating the `/dist/testkit/puppeteer'` gets confused and exports both methods instead of a single one

I console-logged this line and got `load(require('../src/RadioGroup/RadioButton/RadioButton.uni.driver'))` 

```js
{
  createRadioButtonsGetter: [Function: createRadioButtonsGetter],
  radioGroupUniDriverFactory: [Function: radioGroupUniDriverFactory]
}
```

After removing the exported `createRadioButtonsGetter`, this is the output
```js
Promise {
  {
    exists: [Function: exists],
    element: [Function: element],
    click: [Function: click],
    selectByValue: [Function: selectByValue],
    selectByIndex: [Function: selectByIndex],
    getRadioValueAt: [Function: getRadioValueAt],
    getRadioAtIndex: [Function: getRadioAtIndex],
    getSelectedValue: [Function: getSelectedValue],
    isRadioDisabled: [Function: isRadioDisabled],
    getNumberOfRadios: [Function: getNumberOfRadios]
  }
}
```

I think that simply extracting this method to some shared file will solve it without any massive refactors.
I'll try doing a quick fix and post here the results

**Comment by shlomitc** [comment]
fixed in 8.41.0
